_posts/2018-02-28-smart-window-shades.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: Smart Window Shades
+date: 2018-03-01 00:00:00
+categories: projects
+description: A web service that allows one to control their window shades using a website
+author: Ayush Goel
+author-image: https://avatars2.githubusercontent.com/u/10407155?s=460&v=4
+author-bio: First Year Computer Science Student at Rochester Institute of Technology
+author-email: axg4975@rit.edu
+author-social:
+  github: https://github.com/axg4975
+  linkedin: https://www.linkedin.com/in/ayush-goel-589ba215a/
+---
+# Summary
+Smart shades is a web service that allows one to control their window shades using a website.
+The hardware requirement is a Raspberry Pi (connected to the Internet), a stepper motor, housing for the stepper motor (3D printed), a stepper motor driver (I used the Pololu A4988), a 10K ohm resistor, and a domain.
+The application uses Python, Flask, OpenShift in the back end, Bootstrap, JavaScript, and Ajax.
+In short, there are two Flask apps running, one on [a CSH domain](shades.csh.rit.edu) and one on the Pi.
+The Flask app on the CSH website has the JavaScript and Ajax to make requests to the Flask app on the Pi, which can then take care of changing the window shades.
+
+# The Story
+I picked up this IOT project from R&D.
```

Comment:
Since these posts are largely meant for people outside of CSH, you may want to clarify this more. Maybe something like `from our R&D Director, Spencer.`, so you aren't just saying his name with no explanation.